opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -269,12 +269,10 @@ namespace Opm
                 return primary_variables_evaluation_[SFrac];
             }
         }
-        else if (FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx) && compIdx == Indices::canonicalToActiveComponentIndex(FluidSystem::waterCompIdx)) {             
-
+        else if (FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx)) {             
             if (FluidSystem::phaseIsActive(FluidSystem::gasPhaseIdx) && compIdx == Indices::canonicalToActiveComponentIndex(FluidSystem::gasCompIdx)) {
                 return primary_variables_evaluation_[GFrac];
             }
```

Comment:
I was about to comment on this: Should there not be a way to return `primary_variables_evaluation_[WFrac]` here? I.e. the line deleted below should have been kept? @totto82 please comment since you merged.

Reply:
I thought so too. But after carefully reading I concluded that this was correct. The WFRAC is not used for gas-water simulations. The water fraction is 1-primary_variables_evaluation(gfrac) See below

Reply:
Thanks. I see it now.